tests/cases/conformance/es6/destructuring/emptyVariableDeclarationBindingPatterns02_ES5.ts

```diff
@@ -0,0 +1,12 @@
+// @target: es5
+// @declaration: true

```

Comment:
I've already written tests for that elsewhere. See

* `tests/cases/conformance/es6/destructuring/emptyAssignmentPatterns01_ES5.ts`
* `tests/cases/conformance/es6/destructuring/emptyObjectBindingPatternParameter01.ts`
* `tests/cases/conformance/es6/destructuring/emptyArrayBindingPatternParameter01.ts`